src/dropdown/dropdown.ts

```diff
@@ -364,9 +388,18 @@ export class NgbDropdown implements OnInit, OnDestroy {
       * apply the new placement
       * in case of top use up-arrow or down-arrow otherwise
       */
-      const dropdownClass = placement.search('^top') !== -1 ? 'dropup' : 'dropdown';
+      let dropdownClass = placement.search('^top') !== -1 ? 'dropup' : 'dropdown';
       renderer.addClass(dropdownElement, dropdownClass);
 
+      // Add classes to the menu
```

Comment:
I think it's mandatory here. This class is used to place the menu 'right' align, provided by bootstrap: https://getbootstrap.com/docs/4.2/components/dropdowns/#menu-alignment

But while this is done automatically when the position is calculated, the application must be able to do it in static cases. The more obvious example is when the navigation menu is placed on the right side.

Reply:
Finally, it's maybe not a good idea (to add the dropdown-menu-right class), because of the different placement possibilities proposed by Bootstrap, in terms of alignment. See 'Responsive alignment' in the 'menu alignment section' of the Bootstrap doc. 

So it's useless to automatically ad the class, when it's static, the developer will have to do it by himself.